### DISCUSSION TITLE
Installation/usage best practices

### DISCUSSION THREAD

**Discussion by julienrott** [jhipster/generator-jhipster-quarkus#286] [Q&A] [answered]
Hi,

In the documentation on the JHipster site, it is said that we need to install JHipster first (https://www.jhipster.tech/blueprints/quarkus/001_installing_jhipster_quarkus.html) and at the same time that we don't have to install it as it is embedded (https://www.jhipster.tech/blueprints/quarkus/002_creating_an_application.html).

What is the best way to use this blueprint ?

Especially now, as JHipster v8 is being released, is this blueprint ready to use it ?

Regards.

**Comment by mraible** [comment]
If you install JHipster first, you'll need to install version 7.9.3. This blueprint hasn't been updated for JHipster 8.

**Comment by vishal423** [comment, marked as answer]
I suggest to always use `jhipster-quarkus` CLI as documented under [creating an application](https://www.jhipster.tech/blueprints/quarkus/002_creating_an_application.html) section to avoid any compatibility issues.

**Comment by julienrott** [comment]
Thanks for your answers, I’ll go with the `jhipster-quarkus` CLI.
